server/aws/src/.env.example

```
@@ -28,6 +28,8 @@ AUTHZ_BASE_URL=http://localhost:8082
 AUTH_SERVERS_TABLE_NAME=ResourcesStack-DataStoresAuthServersTableXXXXXXXX-XXXXXXXXXXXX
 # Required for DynamoDB authz OAuth client store (deploy output: AuthzOAuthClientsTableName)
 AUTHZ_OAUTH_CLIENTS_TABLE_NAME=ResourcesStack-DataStoresAuthzOAuthClientsTableXXXXXXXX-XXXXXXXXXXXX
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTHZ_OAUTH_CLIENTS_TABLE_NAME key should go before the AUTH_SERVERS_TABLE_NAME key

(UnorderedKey)